appveyor.yml

```diff
@@ -72,6 +72,8 @@ install:
   - git clone https://github.com/alex85k/wingetopt -b %WINGETOPT_VER% C:\wingetopt
   - python -V
   - python -m pip install psycopg2
+  - ps: Invoke-Webrequest 'https://framagit.org/fperrad/lua-MessagePack/raw/master/src5.3/MessagePack.lua?inline=false' -OutFile c:\lua\MessagePack.lua
```

Comment:
Are there any files added by MessagePack that need to included in the zip file we build towards the end of the process?

Reply:
Well, the flex backend in osm2pgsql will not work without MessagePack available. Whether that means that we want to include it in a zip file, I don't know. osm2pgsql probably wouldn't find it anyway, because it depends on the Lua path.

Reply:
MessagePack seems to be a single lua file only, but the relevant version also depends on LuaJIT (the 5.3 version probably won’t run in that case). I think for the time being, having some description where to download MessagePack and how to set lua path would be a good.

Reply:
The newest version of this PR doesn't need MessagePack any more.